metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -426,7 +426,7 @@ def _create_lineage_map(self, entries: Iterable[QueryEvent]) -> Dict[str, Set[st
                 if ref_view_str != destination_table_str:
                     lineage_map[destination_table_str].add(ref_view_str)
                     has_view = True
-            if self.config.lineage_use_sql_parser and has_table and has_view:
+            if self.config.lineage_use_sql_parser and (has_table or has_view):
```

Comment:
Please, can you revert this as SQL parser only needs if we have a table and view dependency as well otherwise, we don't need the parser to decide if the table is only there because of the view reference it but the query not.

Reply:
Oke, that makes sense! Ill push the changes later today.

Reply:
@treff7es done :heavy_check_mark: